### PR TITLE
Student ui

### PR DESCRIPTION
**Pull request by KfirMor**
UI changes:
1. Added new images for Tinti in the login page and Student details (add/delete)
2. Updated the layout of the login page (removed some of the space on top)
3. Added the icons for male and female in the student details pages
4. Replaced the images and links in the top navbar with updated image and text of the project
5. Slightly updated the layouts of the student details page (margins and paddings)

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
+import { DomSanitizer } from '@angular/platform-browser';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss'],
 })
-export class AppComponent {}
+export class AppComponent {
+  constructor(private matIconRegistry: MatIconRegistry, private domSanitizer: DomSanitizer) {
```
Comment (eran1205) [inline src/app/app.component.ts]: Hi dude,

Why this code in the constructor? Aren't the SVG's behave as regular images and can be set in the assets folder?

File: src/app/pages/login/login.component.html [hunk 2]
```diff
@@ -1,8 +1,11 @@
 <div class="login-container">
-    <form name="form" (ngSubmit)="f.form.valid && login()" #f="ngForm" novalidate>
+    <div class ="msw-login-header">
+        <img src="/assets/img/school-logo.png" alt="" align="right">ביה"ס יחדיו
+      </div>
+        <form name="form" (ngSubmit)="f.form.valid && login()" #f="ngForm" novalidate>
           <div class="login-card">
             <div class="msw-login-logo-ph">
-              <img src="/assets/img/msw-proj-logo.png" alt="" width="80" height="80">
+              <img src="/assets/img/msw-proj-logo.png" alt="" width="80" height="80" style="background-color:white;">
```
Comment (eran1205) [inline src/app/pages/login/login.component.html]: Can we move it to be in the SCSS file as a class?

File: src/app/pages/login/login.component.html [hunk 3]
```diff
@@ -36,9 +39,11 @@
...
             </div>
-
-          </div>
-
-    </form>
+            </div>
+        </form>
+        <div class ="msw-login-footer" >
+            <img src="/assets/img/tinti.png" alt="" valign="bottom" style="margin-right: 25%;">
+            <div style="background-color: lightgray; height:48.2px;"></div>
```
Comment (eran1205) [inline src/app/pages/login/login.component.html]: Please move it to scss class.

File: src/app/pages/login/login.component.scss [hunk 4]
```diff
@@ -80,6 +81,28 @@
...
+  letter-spacing: 0.4px;
+  color: #222222;
+  text-align: right;
+  line-height: 80px;
+}
+
+.msw-login-footer {
+  margin-top: 36px;
+  vertical-align: baseline;
+  //  font-size: 21px;
```
Comment (eran1205) [inline src/app/pages/login/login.component.scss]: Can you delete the comment? (we have source control...)

File: src/app/pages/student/details/tabs/student-details-personal-info/student-details-personal-info.component.html [hunk 5]
```diff
@@ -4,7 +4,7 @@
   </div>
   <div class="msw-student-body">
     <div class="msw-student-user-pic-ph">
-      <img src="http://via.placeholder.com/140x100">
+      <img src="/assets/img/picking.tinti.png" alt="" width="80" height="80" style="background-color:white;">
```
Comment (eran1205) [inline src/app/pages/student/details/tabs/student-details-personal-info/student-details-personal-info.component.html]: Please move the styles to SCSS file...

File: src/app/app.component.ts (hunk 1 above)
Comment (KfirMor) [reply src/app/app.component.ts]: The SVGs are not standard graphical format as they include code.. Yes, they are located in the assets folder.
the main issues is that in order to use them in Angular Materials, you need to load them in advance. Product asked to override the default icons for the male/female in the dialog of the student details. This is the way to do it.. if you want to move this code elsewhere, NP. let me know where and I'll move.

Thanks, Kfir

File: src/app/pages/login/login.component.html (hunk 2 above)
Comment (KfirMor) [reply src/app/pages/login/login.component.html]: Done

File: src/app/pages/login/login.component.html (hunk 3 above)
Comment (KfirMor) [reply src/app/pages/login/login.component.html]: Done

File: src/app/pages/student/details/tabs/student-details-personal-info/student-details-personal-info.component.html (hunk 5 above)
Comment (KfirMor) [reply src/app/pages/student/details/tabs/student-details-personal-info/student-details-personal-info.component.html]: Done

File: src/app/pages/login/login.component.scss (hunk 4 above)
Comment (KfirMor) [reply src/app/pages/login/login.component.scss]: Done

File: src/app/app.component.ts (hunk 1 above)
Comment (eran1205) [reply src/app/app.component.ts]: I think that we need to put a function in the OnInit and to initialise these icons there. What do you think?

File: src/app/pages/login/login.component.html [hunk 6]
```diff
@@ -36,9 +38,11 @@
...
               <!--<button mat-button (click)="loginBtn()">הרשמה</button>-->
             </div>
-
-          </div>
-
-    </form>
+            </div>
+        </form>
+        <div class ="msw-login-footer" >
+            <img src="/assets/img/tinti.png" alt="" valign="bottom" style="margin-right: 25%;">
```
Comment (eran1205) [inline src/app/pages/login/login.component.html]: Can you move it to css class?

File: src/app/pages/student/details/tabs/student-details-personal-info/student-details-personal-info.component.html (hunk 5 above)
Comment (eran1205) [inline src/app/pages/student/details/tabs/student-details-personal-info/student-details-personal-info.component.html]: Can you move it to css class?

Comment (KfirMor) [reply src/app/pages/student/details/tabs/student-details-personal-info/student-details-personal-info.component.html]: Already moved it, I guess I forgot to delete from the HTML.............. did it now

File: src/app/pages/login/login.component.html (hunk 6 above)
Comment (KfirMor) [reply src/app/pages/login/login.component.html]: Fadiha. done.

File: src/app/pages/student/details/tabs/student-details-personal-info/student-details-personal-info.component.html (hunk 5 above)
Comment (KfirMor) [reply src/app/pages/student/details/tabs/student-details-personal-info/student-details-personal-info.component.html]: Done.

File: src/app/pages/login/login.component.html (hunk 6 above)
Comment (KfirMor) [reply src/app/pages/login/login.component.html]: Done. Fadiha.

File: src/app/app.component.ts (hunk 1 above)
Comment (KfirMor) [reply src/app/app.component.ts]: Done. I tried to put it on the OnInit of the StudentDetails.. page and it didn;t work. So I klept it on the AppComponent but removed it from the Constructor into the ngOnInit.

**Comment by eran1205** [comment]
Hi Kfir,

The coverage went down...Can you fix?

**Pull request closed without merging**